### PR TITLE
callSafely utility

### PR DESCRIPTION
**Pull request by itamarst**
Catch exceptions since Quark code can't handle them.

### CODE REVIEW

File: quarkc/lib/quark_threaded_runtime.py [hunk 1]
```diff
@@ -526,6 +526,14 @@ def sleep(self, seconds):
     def uuid(self):
         return str(uuid.uuid4())
 
+    def callSafely(self, unary_callable, default):
+        try:
+            return quark.callUnaryCallable(unary_callable, None)
+        except:
```
Comment (ark3) [inline quarkc/lib/quark_threaded_runtime.py]: Why not `except Exception:` here?

File: quarkc/lib/quark.q [hunk 2]
```diff
@@ -109,6 +109,14 @@ interface Runtime {
 
     @doc("Get a v4 random UUID (Universally Unique IDentifier)")
     String uuid();
+
+    @doc("""
+    Call a UnaryCallable safely, catching native exceptions.
+
+    The result of calling the UnaryCallable will be returned, unless an
+    exception is caught in which case the default is returned.
+    """)
```
Comment (ark3) [inline quarkc/lib/quark.q]: Consider adding "The callable is called with null as the argument." or similar.

File: quarkc/lib/quark_threaded_runtime.py (hunk 1 above)
Comment (itamarst) [inline quarkc/lib/quark_threaded_runtime.py]: They're equivalent, there's no real difference between them.

File: quarkc/lib/quark.q (hunk 2 above)
Comment (itamarst) [inline quarkc/lib/quark.q]: Done.

**Pull request merged**